lib/belay_brokerage.ex

```diff
@@ -2,73 +2,83 @@ defmodule BelayBrokerage do
   @moduledoc """
   BelayBrokerage provides an interface against the BelayBrokerage defined DB
   """
-  alias BelayBrokerage.Auth0Id
+  alias BelayBrokerage.AuthAccount
   alias BelayBrokerage.Transactions
   alias BelayBrokerage.Investor
   alias BelayBrokerage.Holding
   alias BelayBrokerage.Repo
 
   import Ecto.Query
-  import Ecto.Changeset
 
   require Logger
 
   @type investor :: %{
-          required(:id) => String.t(),
           required(:first_name) => String.t(),
           required(:last_name) => String.t(),
           required(:address_1) => String.t(),
           optional(:address_2) => String.t(),
           required(:city) => String.t(),
           required(:region) => String.t(),
           required(:postal_code) => String.t(),
-          required(:email) => String.t(),
           required(:phone) => String.t(),
           required(:access_token) => String.t(),
-          required(:account_id) => String.t(),
-          required(:auth0_ids) => [String.t()],
-          required(:item_id) => String.t(),
-          required(:dwolla_customer_id) => String.t()
+          optional(:account_id) => String.t(),
+          optional(:auth_accounts) => [AuthAccount.t()],
+          optional(:item_id) => String.t(),
```

Comment:
I do wish I knew what an item_id was

Reply:
It's how plaid identifies a "plaid link"

Reply:
Can we name it? item_id is so generic it's a little hard to follow

Reply:
Sure, I'll prefix them with plaid